Labelbox/labelbox-python
# [PTDT-1112] Add delete_unused_feature_schema method

- calls a new REST endpoint, what should help with SDK4.0 migration
- delete_unused_feature_schema can be used to delete a feature schema

### How to test

If https://github.com/Labelbox/intelligence/pull/13388 is not merged, git checkout lgluszek/PTDT-1112 in `intelligence`

Usage:

```python
from labelbox import Client

client = Client(
    api_key="...",
    rest_endpoint="http://localhost:3000/api/api/v1")

res = client.delete_unused_feature_schema("...")
```

There should be errors when:

- feature schema is already deleted
- you try to remove a feature schema that is from a different organization
- a feature schema is used in annotations
- a feature schema is added to ontologies
